flash/data/process.py

```diff
@@ -101,6 +101,151 @@ class PreprocessState:
 
 
 class Preprocess(Properties, torch.nn.Module):
+    """
+    The :class:`~flash.data.process.Preprocess` is used to encapsulate
+    all the processing data loading logic up to the model.
+
+    It is particularly relevant when you want to provide ready-to-production implementation which works
```

Comment:
This sounds better to me

```suggestion
    It is particularly relevant when you want to provide an end to end implementation which works
```

Reply:
Done !